### DISCUSSION TITLE
PHP Version

### DISCUSSION THREAD

**Discussion by muriloperosa** [muriloperosa/discord-webhook#11] [Q&A] [answered]
Hey everyone,

We're discussing the PHP versions we'd like to support. We want to hear everyone's thoughts!

If you have any preferences or recommendations regarding the PHP versions we should support, please share them in the comments below. Your input is highly valuable to us!

**Comment by jrfernandodasilva** [comment, marked as answer]
If possible, adding php 5 compatibility, for legacy projects, would be awesome.
